src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/CreateInductionTest.kt

```diff
@@ -110,32 +110,30 @@ class CreateInductionTest : IntegrationTestBase() {
   }
 
   @Test
-  fun `should create induction but fail to create timeline event given timeline database exception`() {
```

Comment:
This was the original test we wrote. Whilst it didnt address the validation issue, it did ensure that timeline events were written to the database asynchronously, and that any errors were not thrown back to the main request thread.

Now that we have validation in place (via the changes in the swagger spec) the test as written fails, and has been rewritten to assert a 400 BAD_REQUEST

I see no value in implementing this same test for all the API endpoints that accept a `prisonId` in the request body (otherwise we'd be writing "request validation" tests for every field in every API request)